test/s1/appframework/appframework_test.go

```diff
@@ -145,4 +145,71 @@ var _ = Describe("s1appfw test", func() {
+			// Upload ES app to S3
+			esApp := "SplunkEnterpriseSecuritySuite"
+			appListV1 = append(testenv.BasicApps, esApp)
+			appFileList := testenv.GetAppFileList(appListV1, 1)
+
+			// Download V1 Apps from S3
+			err := testenv.DownloadFilesFromS3(testDataS3Bucket, s3AppDirV1, downloadDirV1, appFileList)
+			Expect(err).To(Succeed(), "Unable to download V1 app files")
+
+			// Upload V1 apps to S3
+			uploadedFiles, err := testenv.UploadFilesToS3(testS3Bucket, s3TestDir, appFileList, downloadDirV1)
+			Expect(err).To(Succeed(), "Unable to upload ES app to S3 test directory")
+			uploadedApps = append(uploadedApps, uploadedFiles...)
+
+			// Create App framework Spec
+			volumeName := "appframework-test-volume-" + testenv.RandomDNSName(3)
+			volumeSpec := []enterprisev1.VolumeSpec{testenv.GenerateIndexVolumeSpec(volumeName, testenv.GetS3Endpoint(), testenvInstance.GetIndexSecretName(), "aws", "s3")}
+
+			appSourceDefaultSpec := enterprisev1.AppSourceDefaultSpec{
+				VolName: volumeName,
+				Scope:   "local",
+			}
+			appSourceName := "appframework-" + testenv.RandomDNSName(3)
+			appSourceSpec := []enterprisev1.AppSourceSpec{testenv.GenerateAppSourceSpec(appSourceName, s3TestDir, appSourceDefaultSpec)}
+			appFrameworkSpec := enterprisev1.AppFrameworkSpec{
+				Defaults:             appSourceDefaultSpec,
+				AppsRepoPollInterval: 60,
+				VolList:              volumeSpec,
+				AppSources:           appSourceSpec,
+			}
+
+			// Create Standalone spec with App Framework enabled and some extra config to have ES installed correctly
+			spec := enterprisev1.StandaloneSpec{
+				CommonSplunkSpec: enterprisev1.CommonSplunkSpec{
+					Spec: splcommon.Spec{
+						ImagePullPolicy: "Always",
+					},
+					Volumes:                      []corev1.Volume{},
+					LivenessInitialDelaySeconds:  600,
+					ReadinessInitialDelaySeconds: 660,
+				},
+				AppFrameworkConfig: appFrameworkSpec,
+			}
+
+			standalone, err := deployment.DeployStandalonewithGivenSpec(deployment.GetName(), spec)
+			Expect(err).To(Succeed(), "Unable to deploy Standalone with App framework")
+
+			// Ensure Standalone goes to Ready phase
+			testenv.StandaloneReady(deployment, deployment.GetName(), standalone, testenvInstance)
+
+			// Verify Apps are downloaded by init-container
+			initContDownloadLocation := "/init-apps/" + appSourceName
+			podName := fmt.Sprintf(testenv.StandalonePod, deployment.GetName(), 0)
+			appFileList = testenv.GetAppFileList(appListV1, 1)
+			testenv.VerifyAppsDownloadedByInitContainer(deployment, testenvInstance, testenvInstance.GetName(), []string{podName}, appFileList, initContDownloadLocation)
+
+			// Verify ES app is installed locally
+			standalonePod := []string{fmt.Sprintf(testenv.StandalonePod, deployment.GetName(), 0)}
+			testenv.VerifyAppInstalled(deployment, testenvInstance, testenvInstance.GetName(), standalonePod, appListV1, false, "enabled", false, false)
```

Comment:
Nit: `standalonePod` can be replaced by `[]string{fmt.Sprintf(testenv.StandalonePod, deployment.GetName(), 0)}`

Reply:
You mean like on line 210? Remove line 210 and directly put `[]string{fmt.Sprintf(testenv.StandalonePod, deployment.GetName(), 0)}` into line 211?

Reply:
Yeap you can although its NIT so totally upto you.